sources/packages/api/src/types/userInfo.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserInfo {
+  lastName: string;
+  givenNames: string;
+  email: string;
+  userName: string;
```

Comment:
May help to describe here that this is DIDand then @bcsc

Reply:
Yes